render.yaml

```diff
@@ -6,12 +6,18 @@ services:
     repo: https://github.com/BenMemberg/how2meet.git
     region: ohio
     plan: free
-    branch: main
-    rootDir: how2meet
+    branch: tas-115_render_deploy
```

Comment:
We'll want to update this to either main or a deploy branch before merging I imagine?

Reply:
Yep. this is why I asked that question in Slack. I'll update to `main` for now